File: ShadhinMusicLibrary/src/main/java/com/shadhinmusiclibrary/data/model/lastfm/Similar.java
package com.shadhinmusiclibrary.data.model.lastfm;

import androidx.annotation.Keep;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

import java.util.List;

@Keep
public class Similar {
    @SerializedName("artist")
    @Expose
    private List<LastFmArtist> artist = null;

    public List<LastFmArtist> getArtist() {
        return artist;
    }

    public void setArtist(List<LastFmArtist> artist) {
        this.artist = artist;
    }
}

File: ShadhinMusicLibrary/src/main/java/com/shadhinmusiclibrary/data/model/lastfm/Similar.kt
package com.shadhinmusiclibrary.data.model.lastfm

import androidx.annotation.Keep
import com.google.gson.annotations.SerializedName
import com.google.gson.annotations.Expose
import com.shadhinmusiclibrary.data.model.lastfm.LastFmArtist

@Keep
internal class Similar {
    @SerializedName("artist")
    @Expose
    var artist: List<LastFmArtist>? = null
}